packages/create-jest/bin/create-jest.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const importLocal = require('import-local');
```

Comment:
Hm.. `import-local` is not included in `dependencies` list?

Reply:
Also not so sure if `import-local` is needed in this case. It does not make sense installing `create-jest` locally. To be honest, I don’t understand what `import-local` does ;D So it can be I missed something.

Reply:
I'm ok with removing that, local installation indeed seems strange in this case